src/Megadraft.js

```diff
@@ -37,6 +39,9 @@ export default class Megadraft extends Component {
     this.plugins = this.props.plugins || DEFAULT_PLUGINS;
 
     this.keyBindings = this.props.keyBindings || [];
+
+    this.mediaDefaultFeatured = this.props.mediaDefaultFeatured || "medium";
```

Comment:
Could we move this default option to a constant and import it from the base plugin?